azure-proto-compute/ComputeApiVersions.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace azure_proto_compute
+{
+    public class ComputeRestVersions
```

Comment:
```suggestion
    public class ComputeRestApiVersions
```

Reply:
Or perhaps just matching the file name and calling this type `ComputeApiVersions`